src/modules/auth/credentials/FormFields.js

```diff
@@ -52,16 +48,7 @@ export function EmailField(props) {
 	const { t } = useTranslation();
 	const disable = props.disable == undefined ? false : props.disable;
 	const reg = props.register(
-		"email", {
-			required: props.required ? t("FormFields|Email cannot be empty!") : false,
-			validate: {
-				emptyInput: value => (
-					props.getValues("phone") === undefined ?
-					props.getValues("email") !== "" :
-					props.getValues("phone") !== "" || props.getValues("email") !== "") || t("FormFields|Email cannot be empty!"),
-			}
-		}
-	);
+		"email", { required: false, pattern:{value: /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/u , message: "Please enter the correct email"}});
```

Comment:
@Samsunggora Why you set required to `false` ? It is required when the particular prop is passed to it. Also, this email and phone field is used in different screens (when creating/inviting a user) - and I have a feeling, that since you removed some validation on empty content and the required prop, that it will not work as it should in other screens, where email and phone fields are used.

Reply:
> @SamsunggoraПочему вы установили обязательно `false`? Это требуется, когда конкретная опора передается ему. Кроме того, это поле электронной почты и телефона используется на разных экранах (при создании/приглашении пользователя) - и у меня есть ощущение, что, поскольку вы удалили некоторую проверку пустого содержимого и требуемой поддержки, это не будет работать должным образом в другие экраны, где используются поля электронной почты и телефона.

I think that the logic of work on different screens, different, I can rewrite the functionality of the Create new credentials button so that everything works correctly.

Reply:
@Samsunggora Well, you should test and eventually refactor

Reply:
@Samsunggora any update here? ^^^

It does not seem alright as my above question